src/VerifyOTP.tsx

```diff
@@ -115,9 +116,13 @@ export default class VerifyOTP extends React.PureComponent<Props, State> {
         countDownSecond: prevState.countDownSecond - 1,
       }),
       () => {
+        const { onCountDownFinished } = this.props;
         if (this.state.countDownSecond === 0 && this.timerId) {
           clearInterval(this.timerId);
           this.timerId = null;
+          if (onCountDownFinished != null) {
+            onCountDownFinished();
+          }
```

Comment:
@clarkwkw 
How would you change back the count down text styles if it start count down again?

Reply:
It seems the library user could only restart the timer by the callback in `onPressResend?: (restartTimer: () => void) => void;`. So my original plan is to let the library user keep a boolean `isCountDownFinished` and use the `onCountDownFinished` callback to keep track of it.